sonic_installer/common.py

```diff
@@ -23,3 +23,19 @@ def run_command(command):
 
     if proc.returncode != 0:
         sys.exit(proc.returncode)
+
+# Run bash command, print output to stdout and return output on success
+def run_command_output(command):
+    click.echo(click.style("Command: ", fg='cyan') + click.style(command, fg='green'))
+
+    proc = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
+    (out, _) = proc.communicate()
+
+    click.echo(out)
+
+    if proc.returncode != 0:
+        click.echo(click.style("FAILED!", fg='red'))
+        sys.exit(proc.returncode)
+
+    return out
```

Comment:
Try reuse run_command()?
If you don't need `FAILED!` in output, suggest you just rewrite run_command().